site/public/js/forum.js

```diff
@@ -1283,6 +1284,81 @@ function modifyThreadList(currentThreadId, currentCategoriesId, course, loadFirs
     });
 }
 
+function toggleLike(post_id, thread_id, current_user) {
+
+    // eslint-disable-next-line no-undef
```

Comment:
Remove eslint ignores

Reply:
I think I need it there. I checked other functions and every buildCourseUrl has this eslint ignore before they call the buildCourseUrl.